### DISCUSSION TITLE
Route [admin.servergateway.index] is not defined` error by creating CRUD with Tomato/PHP

### DISCUSSION THREAD

**Discussion by chegmarco1989** [tomatophp/tomato#27] [Q&A]
Hello.

**1 -** Unable to generate CRUD. When I try to generate my first CRUD by following this link: https://docs.tomatophp.com/get-started/start-your-first-crud. 
But After create and make `servergateway` migration, I add in `app\Providers\AppServiceProvier`, this following in its `boot` method:

```
use TomatoPHP\TomatoAdmin\Facade\TomatoMenu;
use TomatoPHP\TomatoAdmin\Services\Contracts\Menu;

... 

public function boot()
{
    TomatoMenu::register(
        Menu::make()
            ->group('Resources')
            ->label('Users')
            ->route('admin.servergateway.index')
            ->icon('bx bx-user')
    );
}
```

I get `Route [admin.servergateway.index] is not defined` error.

**Does Documentation about How to create a first CRUD is really complete ???** 

Otherwise, please help me to fix that error and succeed my CRUD's creation. 

**2 -** Do I need to creed my own View or the generator create it and its controller for me automatically ???

**3 -** If the generator create it for me automativally, how to edit them if I need ??? How to modify that generated View and Controller if I want ???

Awaiting your answers.
Thanks.

**Comment by fadymondy** [comment]
hi @chegmarco1989 sorry for this time, your problem is just registering the route it self on your `web.php`
